Makefile

```diff
@@ -19,8 +19,9 @@ lint-local:
 	golangci-lint run ./...
 
 .PHONY: lint
-lint:
-	exit
+lint: generate-prod
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.45.2
```

Comment:
We can use an upgraded version of golangci-lint now the app has been upgraded to go 1.19